website/src/react-native/showcase.js

```diff
@@ -5,1142 +5,292 @@
     linkPlayStore: 'https://play.google.com/store/apps/details?id=com.facebook.adsmanager',
-    author: 'Facebook',
-    blogs: [
-      "https://code.facebook.com/posts/1014532261909640/react-native-bringing-modern-web-techniques-to-mobile/",
-      "https://code.facebook.com/posts/1189117404435352/react-native-for-android-how-we-built-the-first-cross-platform-react-native-app/",
-      "https://code.facebook.com/posts/435862739941212/making-react-native-apps-accessible/",
-    ],
+    infoLink: 'https://code.facebook.com/posts/1189117404435352/react-native-for-android-how-we-built-the-first-cross-platform-react-native-app/',
+    infoTitle: 'How We Built the First Cross-Platform React Native App',
   },
   {
     name: 'F8',
     icon: 'https://raw.githubusercontent.com/fbsamples/f8app/master/ios/F8v2/Images.xcassets/AppIcon.appiconset/AppIcon%402x.png',
     linkAppStore: 'https://itunes.apple.com/us/app/f8/id853467066?mt=8',
     linkPlayStore: 'https://play.google.com/store/apps/details?id=com.facebook.f8',
-    author: 'Facebook',
-    blogs: [
-      "http://makeitopen.com/tutorials/building-the-f8-app/planning/",
-      "http://makeitopen.com/tutorials/building-the-f8-app/design/",
-      "http://makeitopen.com/tutorials/building-the-f8-app/data/",
-      "http://makeitopen.com/tutorials/building-the-f8-app/testing/",
-    ],
-  },
-  {
-    name: 'AIGA Design Conference 2015',
-    icon: 'http://a5.mzstatic.com/us/r30/Purple69/v4/b0/4b/29/b04b2939-88d2-f61f-dec9-24fae083d8b3/icon175x175.png',
-    link: 'https://itunes.apple.com/us/app/aiga-design-conference-2015/id1038145272?ls=1&mt=8',
-    author: 'W&Co',
+    infoLink: 'http://makeitopen.com/tutorials/building-the-f8-app/planning/',
+    infoTitle: 'Building the F8 App',
   },
   {
     name: 'Discord',
     icon: 'http://a5.mzstatic.com/us/r30/Purple5/v4/c1/2f/4c/c12f4cba-1d9a-f6bf-2240-04085d3470ec/icon175x175.jpeg',
-    link:  'https://itunes.apple.com/us/app/discord-chat-for-gamers/id985746746?mt=8',
-    author: 'Hammer & Chisel',
-    blogs: [
-      "https://discord.engineering/react-native-deep-dive-91fd5e949933#.5jnqftgof",
-    ],
+    linkAppStore:  'https://itunes.apple.com/us/app/discord-chat-for-gamers/id985746746?mt=8',
+    infoLink: 'https://discord.engineering/react-native-deep-dive-91fd5e949933#.5jnqftgof',
+    infoTitle: 'Using React Native: One Year Later',
   },
   {
     name: 'Discovery VR',
     icon: 'http://a2.mzstatic.com/us/r30/Purple6/v4/d1/d5/f4/d1d5f437-9f6b-b5aa-5fe7-47bd19f934bf/icon175x175.png',
     linkAppStore: 'https://itunes.apple.com/us/app/discovery-vr/id1030815031?mt=8',
     linkPlayStore: 'https://play.google.com/store/apps/details?id=com.discovery.DiscoveryVR',
     author: 'Discovery Communications',
-    blog: [
-      "https://medium.com/ios-os-x-development/an-ios-developer-on-react-native-1f24786c29f0",
-    ],
-  },
-  {
-    name: 'Exponent',
```

Comment:
Well I did put li.st back in the... list. For apps in here I would like to link to something that talks about the app. The exponent app itself, I can't quite find anything that talks about that app. Honestly it is somewhat confusing to me, is it more an app or is it more a development tool? For development tools, our plan during this lockdown week is to have a tab centered on third-party development tools, "More Resources", and to link to development tools there, and then exponent seems great. I think when you are looking for "what cool apps use React Native" you are more in a mode of "let me figure out if this whole thing is right for me" whereas if you are looking for development tools you aren't going to look in the showcase.